### ISSUE TITLE
[MAS1.3.1][Screen Reader-Connect the bot] Voiceover announces the same name on the two control with different role.

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1909]
**Actual Result:**
The voiceover announces the same name i.e. "Restart Conversation" on "Restart conversation" button and "Restart conversation" dropdown.

**Expected Result:**
Both the controls should be associated and consolidated focus should land on them.
Else the voiceover should announce them as "Restart Conversation" menu pop-up combo box.

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Welcome tab screen gets opened.
3. Navigate to the "Open Bot" button and activate it.
4. Open a Bot Dialog box gets opened.
5. Navigate to the "bot URL" edit box and enter URL- http://localhost:3978/api/messages
6. Navigate to "Connect" button and activate it.
7. "Live Chat" separate window appears.
8. Navigate to "Menu Bar" control.
9. Observe the issue.

**User Impact:**
Users who depends on the Screen reader will get confused if same name is defined for both the controls with different role.

**Test Environment:**
OS: MacOs Version 10.15
Application: Bot Framework V4 Emulator (Version 4.5.2-80829)

**Impacted MAS Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}](url)

**Attachment:**
[Same name defined for two controls.zip](https://github.com/microsoft/BotFramework-Emulator/files/3690674/Same.name.defined.for.two.controls.zip)

**Comment by ShikhaMishra11** [comment]
As we have checked and verified, the issue gets fixed in the latest version of BOT. So, we are closing the issue.

Regression Environment:
OS: MacOs Version 10.15
Application: Bot Framework V4 Emulator (Version 4.6.0-93410)
